bulk_indexer.go

```diff
@@ -91,6 +91,12 @@ type BulkIndexerConfig struct {
 	//
 	// RequireDataStream is disabled by default.
 	RequireDataStream bool
+
+	// PopulateFailedDocsSource controls whether each BulkIndexerResponseItem.Source
+	// in BulkIndexerResponseStat.FailedDocs is populated with the source of the item,
+	// which includes the action line and the document line.
+	// For testing and debugging only. Use with caution.
+	PopulateFailedDocsSource bool
```

Comment:
[to reviewer] any suggestions for a better name? is `PopulateFailedDocsBody` clearer?

Reply:
Source might sound too much like `_source`, which is only the document content while in reality we return both action line and document line.

Reply:
I initially assumed it was _source from the PR title, so I think it does sound a bit too much like that. Maybe `PopulateFailedDocsInput`?